### DISCUSSION TITLE
Need to get in touch with a GitHub Security Expert

### DISCUSSION THREAD

**Discussion by realmarkoprifti** [community/community#45706] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

Hello! My name is Marko Prifti and i'm 14 years old. I watched a video in YouTube about how a group named "Purple Urchin", exploited GitHub Actions for cryptomining. I have found a similar bug. Contact me at <Edited by Staff - sensitive personal info>  about the bug.

Marko

**Comment by airtower-luna** [comment]
You might want to take a look at https://github.com/.well-known/security.txt for security reporting (see https://securitytxt.org/ for what the file means). :wink: 

This community forum is mostly for discussions among users, though I've seen staff chip in, too.
